dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ejs/container/util/MethodAttribUtils.java

```diff
@@ -340,6 +341,53 @@ public static boolean getAsynchronousMethods(Method[] beanMethods,
 
     } // getAsynchronousMethods
 
+    /**
+     * <code>getConcurrentAsynchronousMethods<code> checks all beanMethods to determine
+     * if any beans are annotated with the Jakarta Concurrent Asynchronous annotation.
+     * Returns true when found, or false otherwise.
+     *
+     * @param beanMethods Input array of Method objects representing the methods of this EJB.
+     *
+     * @return True if at least one method was found on this bean to have the Concurrent Asynchronous annotation.
+     */
+    public static boolean getConcurrentAsynchronousMethods(Method[] beanMethods) {
```

Comment:
unlike getAsynchronousMethods, I notice this method does not check for the annotation on the Class.  If the annotation can be on the class, is it worth checking?  Could probably add the class as a parameter to this method, and then should be sure to look on super classes as well, up to java.lang.Object.